### PR TITLE
{bio}[intel/2017a] oxford_asl v3.9.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar01.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f0c1a9c518f077c48ee49ebcfcf9233f for a full test report.

**Comment by boegel** [comment]
Note: this is the binary version since several tools seems to be missing when building from source, see https://github.com/ibme-qubic/oxford_asl/issues/3

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 8.8, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/0125f01f53498c6ae5293f035e2e0e69 for a full test report.

File: easybuild/easyconfigs/o/oxford_asl/oxford_asl-3.9.6-intel-2017a-centos7-Python-2.7.13.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+versionsuffix = '-centos7-Python-%(pyver)s'
+
+homepage = 'https://github.com/ibme-qubic/oxford_asl'
+description = "A command line tool for quantification of perfusion from ASL data"
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['https://github.com/ibme-qubic/oxford_asl/files/1292633']
+sources = ['oxford_asl-%(version)s%(versionsuffix)s.tar.gz']
+
```
Comment (vanzod) [inline easybuild/easyconfigs/o/oxford_asl/oxford_asl-3.9.6-intel-2017a-centos7-Python-2.7.13.eb]: Missing checksum

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar01.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/46938cc1744be1ecc8c35efd05b8e6bb for a full test report.

**Comment by boegel** [comment]
@vanzod Good to go?

**Pull request merged**